tests/dialects/test_llvm.py

```diff
@@ -68,16 +68,27 @@ def test_llvm_getelementptr_op_invalid_construction():
 
     # check that passing an opaque pointer to GEP without a pointee type fails
     with pytest.raises(ValueError):
-        llvm.GEPOp.get(opaque_ptr, llvm.LLVMPointerType.typed(builtin.i32),
-                       [1])
+        llvm.GEPOp.get(
+            opaque_ptr,
+            indices=[1],
+            result_type=llvm.LLVMPointerType.typed(builtin.i32),
+        )
 
     # check that non-pointer arguments fail
     with pytest.raises(ValueError):
-        llvm.GEPOp.get(size, llvm.LLVMPointerType.opaque())
+        llvm.GEPOp.get(
+            size,
+            indices=[1],
+            result_type=llvm.LLVMPointerType.opaque(),
+        )
 
     # check that non-pointer result types fail
     with pytest.raises(ValueError):
-        llvm.GEPOp.get(ptr, builtin.i32, [1])  #type: ignore
+        llvm.GEPOp.get(
+            ptr,
+            indices=[1],
+            result_type=builtin.i32,  #type: ignore
```

Comment:
Why the type ignore?

Reply:
Because we are passing in an invalid argument into the function and observing that it indeed errors. Idk how useful that test is but I didn't want to remove the testcase either.

Reply:
:+1: